rpms/README.md

```diff
@@ -5,53 +5,74 @@ needed to build a set of RPMs that are useful for working with Geotrellis.  If
 you are not intending to alter the composition or versions of the RPM
 packages, then you should be able to use the published binaries described in
 the README at the root of this project and do not need to build these
-artifacts yourself. Specifically:
+artifacts yourself.
 
-  > **Do nothing** if you are intending to deploy to EMR and have access to
-  > pre-built RPMs.  Simply point the EMR scripts to the relevant resources on
-  > S3.
+Built RPMs are publicly available at:
 
-  > **Fetch, do not build** if you have access to an already-built set of
-  > RPMs, and your purpose is to use these RPMs to build a Docker container.
+```
+s3://geotrellis-build-artifacts/rpms/
+```
 
-If you have a special use case, and the pre-built binaries are not sufficient,
-then you may need to build them yourself.
+If you need to load them during an EMR bootstrap, you can use `s3://geotrellis-build-artifacts/rpms/bootstrap.sh`.
 
-# Using this Repository #
+For example:
 
-## Fetching ##
+```
+BootstrapAction(
+  "Install GDAL 2.4 dependencies",
+  "s3://geotrellis-build-artifacts/rpms/bootstrap.sh",
+  "s3://geotrellis-build-artifacts", "gdal-2.4.1"
+)
```

Comment:
This block is provided without context, and it's not clear what this is or how to use it.

Reply:
I moved this to a new subsection at the end titled "working with SBT lighter" and referenced it in the first section. We've used both the makefiles and sbt lighter on various projects, figured it was worth documenting that usage here.